ios/Sources/HomeFeature/HomeScreen.swift

```diff
@@ -46,20 +46,24 @@ public struct HomeScreen: View {
                         .separatorStyle(ThickSeparatorStyle())
                     }
                 }
-            }
-            .navigationBarTitleDisplayMode(.inline)
-            .toolbar {
-                ToolbarItem(placement: .principal) {
-                    AssetImage.logoTitle.image
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .principal) {
+                        AssetImage.logoTitle.image
+                    }
+                }
+                .navigationBarItems(
+                    trailing: Button(action: {
+                        viewStore.send(.showSettings)
+                    }) {
```

Comment:
I think this is not solved yet.
https://github.com/DroidKaigi/conference-app-2021/pull/527#discussion_r663434741